_pages/welcome-to-TTS/getting-started.md

```diff
@@ -6,164 +6,164 @@ tags:
-Headers don't work in Slack, but most other Markdown commands do. 
-While you're DM-ing with yourself, you can also click the plus button on the left side of the input box, 
-click "Post", and practice your Markdown in there. 
+There are quite a few online Markdown editors you can use, but after just a little practice you won't need them.
+Pro tip: You can open a DM channel with yourself in Slack and use most Markdown in messages to yourself
+to try it out and see the results.
+Headers don't work in Slack, but most other Markdown commands do.
+While you're DM-ing with yourself, you can also click the plus button on the left side of the input box,
+click "Post", and practice your Markdown in there.
 Slack's posts support more markdown than messages do, and it renders live. Which is pretty neat.
 
 
 ### Security, Compliance, and ATOs
 
-Security and compliance are, unsurprisingly, important issues for TTS. 
-They're also complex topics, and you'll find a lot of people working on them. 
-For an easy introduction into these topics, we suggest you start with 
-[this lightweight primer](https://docs.google.com/presentation/d/1vXZDhEONp-h-zmyCt9wpwscoANouVK4fg5br6Ynzlpk/edit#slide=id.g54fbd835cd_0_25), 
+Security and compliance are, unsurprisingly, important issues for TTS.
+They're also complex topics, and you'll find a lot of people working on them.
+For an easy introduction into these topics, we suggest you start with
+[this lightweight primer](https://docs.google.com/presentation/d/1vXZDhEONp-h-zmyCt9wpwscoANouVK4fg5br6Ynzlpk/edit#slide=id.g54fbd835cd_0_25),
 especially if you haven't worked on information security before. Read the ["Security incidents" page]({{site.baseurl}}/security-incidents/) to learn how to respond if you spot suspicious activity or an incident.
-If you're looking to go into greater detail, 
-18F's [Before You Ship](https://before-you-ship.18f.gov/) guide is definitive and should answer any questions you have, 
-but it can also be a little overwhelming. 
-If you ever need a quick answer, clarification or just general Security and Compliance help, 
-head over to [#wg-security](https://gsa-tts.slack.com/messages/CG8SNAEQM) 
+If you're looking to go into greater detail,
+18F's [Before You Ship](https://before-you-ship.18f.gov/) guide is definitive and should answer any questions you have,
+but it can also be a little overwhelming.
+If you ever need a quick answer, clarification or just general Security and Compliance help,
+head over to [#wg-security](https://gsa-tts.slack.com/messages/CG8SNAEQM)
 or [#wg-compliance](https://gsa-tts.slack.com/messages/C2UUQ5CNP) in Slack.
 
-If you hear "ATO" or "Authority to Operate" and are wondering what that's about, 
-[Before You Ship](https://before-you-ship.18f.gov/) and the Security and Compliance 
+If you hear "ATO" or "Authority to Operate" and are wondering what that's about,
+[Before You Ship](https://before-you-ship.18f.gov/) and the Security and Compliance
```

Comment:
@afeld could before you ship change to the tts domain? If so might want to manage as a config variable?